lib/active_record/tasks/sqlserver_database_tasks.rb

```diff
@@ -59,9 +59,9 @@ def structure_dump(filename, extra_flags)
           "-P #{Shellwords.escape(configuration['password'])}",
           "-o #{Shellwords.escape(filename)}",
         ]
-        table_args = connection.tables.map { |t| Shellwords.escape(t) }
+        table_args = connection.tables.sort.map { |t| Shellwords.escape(t) }
         command.concat(table_args)
-        view_args = connection.views.map { |v| Shellwords.escape(v) }
+        view_args = connection.views.sort.map { |v| Shellwords.escape(v) }
```

Comment:
This one is tricky. What if you have a view that depends on another view but alphabetical order does not match? Wouldn't that be a problem?